hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -281,6 +281,15 @@ def _step_wiping(self, verbose: bool) -> StepRes:
         if self.state.wipe_cycles > 0:
             self.state.wipe_cycles -= 1
 
+        is_good = self.state.get_fsm_state() == FsmState.WIPING_GOOD
+
+        # Zero INSN_CNT if we're in state WIPING_BAD. This is a bit silly
+        # because we'll send that update on every cycle, but it correctly
+        # handles the situation where we switch from WIPING_GOOD to WIPING_BAD
+        # half way through a secure wipe because of an incoming escalation.
+        if not is_good:
+            self.state.ext_regs.write('INSN_CNT', 0, True)
+
```

Comment:
This code is also changed by https://github.com/lowRISC/opentitan/pull/14217, which is now merged. Could you please rebase your PR, so that we can review the changes w.r.t. the current code?